src/test/resources/js/web.tests.js

```diff
@@ -99,3 +99,34 @@ describe("sprat.web.hateoas.embedded", function () {
         expect($hateoas.embedded(object)).toEqual(object._embedded.customers);
     });
 });
+
+describe("sprat.web.hateoas.constraints", function () {
+
+    it("the constraint 'delete' can be found", function () {
+        var object = {
+            constraints: {
+                delete: [
+                    {
+                        id: 111
+                    }
+                ]
+            }
+        };
+
+        expect($hateoas.constraints(object, 'delete')[0]['id']).toEqual(111);
```

Comment:
It should probably check for the actual result of the invoked call.
`var actual = $hateoas.constraints(object, 'delete'); `
`expect(actual).toEqual(expected);`
_expected_ being the delete.id.

IMO the assertion looks a little to complicated.